cinema/serializers.py

```diff
@@ -114,9 +114,9 @@ class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
         data = super(TicketSerializer, self).validate(attrs=attrs)
         Ticket.validate_ticket(
-            attrs["row"], 
-            attrs["seat"], 
-            attrs["movie_session"].cinema_hall, 
+            attrs["row"],
+            attrs["seat"],
+            attrs["movie_session"].cinema_hall,
```

Comment:
In the `validate` method of `TicketSerializer`, ensure that the keys 'row', 'seat', and 'movie_session' exist in the `attrs` dictionary before accessing them. This will prevent potential `KeyError` exceptions if these keys are missing.